### DISCUSSION TITLE
[Bug]: SimpleRestAPi - filter does not work

### DISCUSSION THREAD

**Discussion by Warshoow** [pimcore/pimcore#12259] [General]
### Expected behavior

Hi,
So, based on the documentation on the SimpleRestApi bundle, and the syntax provided here : 

[Documentation SimpleRestApi filtering](https://pimcore.com/docs/data-hub-simple-rest/current/Filtering.html)
[Syntax for filtered request](https://restdb.io/docs/querying-with-the-api)

By making my request with a filter with my parameter on which I want to filter, I'm supposed to get the product I was looking for.

Example: 
`https://domaine-name/pimcore-datahub-webservices/simplerest/rest-config-name/tree-items?type=object&parent_id=4082&include_folders=true&size=200&filter={"unique_code": "12345789"}&include_aggs=false`

### Actual behavior

But the problem is that I get an empty response.
```
{
	"total_count": 0,
	"items": [],
	"labels": []
}
```
I obtain this same result by passing by swagger provided by the bundle SimpleRestApi, on Insomnia or by making Test of request with GuzzleHttp in command php.

Maybe I'm just doing something wrong, and/or something is missing.

### Steps to reproduce

Juste make a request on swagger or any tool like postman.

**Comment by fashxp** [comment]
I'm not able to reproduce it... probably you misspelled the attribute name? make sure you are using `.` notation to specify the attributes group...

like `system.id` or `data.unique_code` ...
